developer/src/tike/compile/Keyman.Developer.System.KmcWrapper.pas

```diff
@@ -50,11 +51,11 @@ function TKmcWrapper.Compile(
   cmdline := Format('"%s" build --log-format tsv --log-level info "%s"', [TKeymanDeveloperPaths.KmcPath, infile]);
   if outfile <> '' then
     cmdline := cmdline + Format(' --out-file "%s"', [outfile]);
-  if ProjectFile.Project.Options.CompilerWarningsAsErrors then
+  if FGlobalProject.Options.CompilerWarningsAsErrors then
```

Comment:
> In 'projectless' mode, there is still a global 'untitled' project, but the files loaded do not 'belong' to it. The compiler settings are inherited from the global 'untitled' project in this case.

Is `FGlobalProject` a superset of "has a .kpj" and `projectless` modes?

Reply:
Yes, `FGlobalProject` is a global singleton, which always points to a valid, in memory project. That can be a project .kpj or the default untitled project for this fallback 'projectless' mode.